snewpdag/plugins/TrueVsFit.py

```diff
@@ -56,15 +56,11 @@ def update(self, data):
     for n,i in enumerate(self.map):
        if(i=='JUNO'):
```

Comment:
This was the line which made me think about how generic the code was.  Do we need to tie these lines to a particular experiment?

Reply:
okey, now I see... 
indeed I did not find an optimal way of implementing this as there are different input fields to define dt1, dt2 and dt_fit.
I have implemented it in a way which makes no reference to detector names, let me know what you think.